src/app/components/TWAP/TWAPOrders.tsx

```diff
@@ -0,0 +1,28 @@
+import { Orders } from '@orbs-network/twap-ui-spiritswap';
+import useWallets from 'app/hooks/useWallets';
+import { useTokens } from 'app/hooks/useTokens';
+import { getProvider } from 'app/connectors/EthersConnector/login';
+import { LimitOrderContainer } from 'app/pages/Swap/components/LimitOrders/styles';
+
+const getTokenImage = (token: any) => {
```

Comment:
hi, you are using  the same four lines of code for `getTokenImage` in `TWAPOrders.tsx` and in `TWAPPanel.tsx`.
Please create a unique component for this or just use or modify `ImageLogo` component, from components folder.